evergreen/system_perf/master/genny_tasks.yml

```diff
@@ -998,6 +1008,14 @@ buildvariants:
 - name: perf-replSet-auth-delay.arm.aws.2023-11
   tasks:
   - name: transport_layer_connect_timing
+- name: perf-1-node-replSet-longRunning.arm.aws.2023-11
+  tasks:
+  - name: device_monitoring
+  - name: non_search_hybrid_scoring
+- name: perf-1-node-replSet-m60-like-large-volume.arm.aws.2023-11
+  tasks:
+  - name: device_monitoring
+  - name: non_search_hybrid_scoring
```

Comment:
I am not sure if we want these to get scheduled here to be honest. We might want to update the corresponding auto-run sections.